recipe/migrations/curl7691.yaml

```diff
@@ -0,0 +1,25 @@
+# To learn more about migrations read CFEP-09
+# https://github.com/conda-forge/conda-forge-enhancement-proposals/blob/master/cfep-09.md
+# The timestamp of when the migration was made
+# Can be obtained by copying the output of
+# python -c "import time; print(f'{time.time():.0f}')"
+migrator_ts: 1586520931
```

Comment:
I'm happy to make the `migrator_ts` more precise (as I have seen in some recent migrators) but the comment in the example should probably be updated to show how this is done.

Reply:
Its fine, the bot uses the raw timestamp from python when it creates these, but it doesn't matter.